pkg/operator/k8sutil/pod.go

```diff
@@ -22,22 +22,40 @@ import (
 	"encoding/json"
 	"fmt"
 	"os"
+	"path"
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/pkg/api"
 	"k8s.io/client-go/pkg/api/v1"
 )
 
 const (
-	AppAttr           = "app"
-	ClusterAttr       = "rook_cluster"
-	VersionAttr       = "rook_version"
-	PodIPEnvVar       = "ROOKD_PRIVATE_IPV4"
-	DefaultRepoPrefix = "quay.io/rook"
-	repoPrefixEnvVar  = "ROOKD_REPO_PREFIX"
-	defaultVersion    = "latest"
+	AppAttr            = "app"
+	ClusterAttr        = "rook_cluster"
+	VersionAttr        = "rook_version"
+	PodIPEnvVar        = "ROOKD_PRIVATE_IPV4"
+	DefaultRepoPrefix  = "quay.io/rook"
+	repoPrefixEnvVar   = "ROOKD_REPO_PREFIX"
+	defaultVersion     = "latest"
+	ConfigOverrideName = "config-override"
```

Comment:
Maybe the name of the ConfigMap should have rook in it, like `rook-config-override`.  If a user chooses not to use a namespace named `rook` then `config-override` becomes ambiguous.